### ISSUE TITLE
ttnn.tan_bw operation fails with low PCC 

### ISSUE THREAD

**Issue by nemanjagrujic** [tenstorrent/tt-metal#13675]
ttnn.tan_bw operation fails with low PCC

 * Steps to reproduce

On branch `ngrujic/unit-tests-1` (soon to be merged into main), there is unit test `tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_tan_bw.py`, which can be run with:

```
pytest  tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_tan_bw.py
```

Also, you can reproduce this error with sweep test `tests/sweep_framework/sweeps/eltwise/unary_backward/tan_bw/tan_bw.py`. To run:

```
python3 tests/sweep_framework/sweeps_parameter_generator.py --elastic cloud --module-name eltwise.unary_backward.tan_bw.tan_bw
python3 tests/sweep_framework/sweeps_runner.py --elastic cloud --module-name eltwise.unary_backward.tan_bw.tan_bw
```

Results can be found on elastic cloud as explained here: `https://github.com/tenstorrent/tt-metal/tree/main/tests/sweep_framework`

**Comment by umadevimcw** [comment]
@nemanjagrujic Can you provide unit test for debugging?

**Comment by nemanjagrujic** [comment]
@umadevimcw , I updated instructions.

**Comment by KalaivaniMCW** [comment]
Hi @nemanjagrujic 

`ttnn.tan_bw` is directly dependent on `ttnn.tan` op for which the currently supported range is (-1.45 to 1.45) hence the same is the case for `ttnn.tan_bw` as well. I'll update the range in sweeps and doc for now. when modulo support is added and range of tan op is expanded we'll retest and update. Is that ok with you ? 

When I tested with the supported range, the sweeps pass
![Image](https://github.com/user-attachments/assets/d39aab01-ea30-4f26-9419-6775fd62bd81)

**Comment by nemanjagrujic** [comment]
@umadevimcw Both docs for tan and tan_bw should be updated then.

**Comment by KalaivaniMCW** [comment]
@nemanjagrujic Yes there is an open issue discussing modulo support in kernel level to increase the range for tan op #6735

**Comment by nemanjagrujic** [comment]
@umadevimcw ok

**Comment by KalaivaniMCW** [comment]
Merged #15119
